### ISSUE TITLE
Extract simple bubbles

### ISSUE THREAD

**Issue by shilpagarg** [vgteam/vg#509]
Is there a way to extract the graph containing simple bubbles only by removing all the nesting?

**Comment by ekg** [comment]
For the application we're thinking about, just determining the outer level of un-nested bubbles would be enough. Would these be the first nodes after the root of the cactus tree? signalling @glennhickey @benedictpaten

**Comment by glennhickey** [comment]
Yeah, that's the idea.  The root's a dummy node and all its children will
be the top level bubbles.  There are a number of fixes on the way in PR
#488 which include not having to define and source and sink.

**Comment by adamnovak** [comment]
I think you can use one of the `SiteFinder`s defined in https://github.com/vgteam/vg/blob/master/src/genotypekit.hpp if you're working inside the vg codebase, or linking against it. They find bubbles, and you could look for bubbles that don't have anything nested inside them and only use those.

Unfortunately I don't think it's exposed anywhere in the command line interface.

**Comment by faithokamoto** [comment]
I think my answer to another issue would also solve this one
